arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedParquetDefinitionLevelReader.java

```diff
@@ -46,122 +47,217 @@ public VectorizedParquetDefinitionLevelReader(
     super(bitWidth, maxDefLevel, readLength, setArrowValidityVector);
   }
 
-  abstract class NumericBaseReader {
-    public void nextBatch(
+  abstract class CommonBaseReader {
```

Comment:
I feel like `Common` and `Base` already try to name the same thing, so maybe just cal this `CommonReader`